fiboa_cli/datasets/sk.py

```diff
@@ -1,76 +1,55 @@
+from ..convert_utils import BaseConverter
 
-SOURCES = {
-    "https://data.slovensko.sk/download?id=e39ad227-1899-4cff-b7c8-734f90aa0b59&blocksize=0": [
-        "HU2024_20240917shp/HU2024_20240917.shp"
-    ]
-}
-ID = "sk"
-SHORT_NAME = "Slovakia"
-TITLE = "Slowakia Agricultural Land Idenfitication System"
-
-DESCRIPTION = """
-Systém identifikácie poľnohospodárskych pozemkov (LPIS)
-
-LPIS is an agricultural land identification system. It represents the vector boundaries of agricultural land
-and carries information about the unique code, acreage, culture/land use, etc., which is used as a reference
-for farmers' applications, for administrative and cross-checks, on-site checks and also checks using remote
-sensing methods.
 
-Dataset Hranice užívania contains the use declared by applicants for direct support.
-"""
-PROVIDERS = [
-    {
-        "name": "National catalog of open data",
-        "url": "https://data.slovensko.sk/",
-        "roles": ["producer", "licensor"]
+class Converter(AdminConverterMixin, BaseConverter):
+    sources = {
+        "https://data.slovensko.sk/download?id=e39ad227-1899-4cff-b7c8-734f90aa0b59&blocksize=0": [
+            "HU2024_20240917shp/HU2024_20240917.shp"
+        ]
     }
-]
-
-LICENSE = "CC-0"  # "Open Data"
-COLUMNS = {
-    "geometry": "geometry",
-    "KODKD": "id",
-    "PLODINA": "crop_name",
-    "KULTURA_NA": "crop_group",
-    "LOKALITA_N": "municipality",
-    "VYMERA": "area",
-}
-COLUMN_MIGRATIONS = {
-    "geometry": lambda col: col.make_valid()
-}
-MISSING_SCHEMAS = {
-    "properties": {
-        "crop_name": {
-            "type": "string"
-        },
-        "crop_group": {
-            "type": "string"
-        },
-        "municipality": {
-            "type": "string"
-        },
+    id = "sk"
+    short_name = "Slovakia"
+    title = "Slovakia Agricultural Land Identification System"
+    description = """
+    Systém identifikácie poľnohospodárskych pozemkov (LPIS)
+
+    LPIS is an agricultural land identification system. It represents the vector boundaries of agricultural land
+    and carries information about the unique code, acreage, culture/land use, etc., which is used as a reference
+    for farmers' applications, for administrative and cross-checks, on-site checks and also checks using remote
+    sensing methods.
+
+    Dataset Hranice užívania contains the use declared by applicants for direct support.
+    """
+    providers = [
+        {
+            "name": "National catalog of open data",
+            "url": "https://data.slovensko.sk/",
+            "roles": ["producer", "licensor"]
+        }
+    ]
+    license = "CC-0"  # "Open Data"
+    columns = {
+        "geometry": "geometry",
+        "KODKD": "id",
+        "PLODINA": "crop_name",
+        "KULTURA_NA": "crop_group",
+        "LOKALITA_N": "municipality",
+        "VYMERA": "area",
+    }
+    column_migrations = {
+        "geometry": lambda col: col.make_valid()
+    }
```

Comment:
Is that still needed? I think we added make_valid to the core, didn't we?

Reply:
Yes, will add a test too...